nixnet/database/_pdu.py

```diff
@@ -54,52 +67,171 @@ def default_payload(self, value):
 
     @property
     def comment(self):
+        # type: () -> typing.Text
+        """str: Get or set a comment describing the PDU object.
+
+        A comment is a string containing up to 65535 characters.
+        """
         return _props.get_pdu_comment(self._handle)
 
     @comment.setter
     def comment(self, value):
+        # type: (typing.Text) -> None
         _props.set_pdu_comment(self._handle, value)
 
     @property
     def config_status(self):
+        # type: () -> int
+        """int: Returns the PDU object configuration status.
+
+        Configuration Status returns an NI-XNET error code.
```

Comment:
this will have to be updated in #233 right?

Reply:
Yes, you are spot on. I expect #233 to go in after this and it should and will be updated with correct annotations/docstrings before it gets pulled.